addons/core/functions/fnc_getSwSettings.sqf

```diff
@@ -65,6 +65,20 @@ if (isNil "_rc") then {
     _value set [TFAR_CODE_OFFSET, _rc];
 };
 
+private _hdo = _value select HALFDUPLEX_OVERRIDE_OFFSET;
```

Comment:
Yeah, I looked around the source to find an alternative to direct config access, since I was trying to avoid it as well. My search was - same as yours - not very successful either...

Will stay as it is for now, until a `getConfigProperty` for `CfgWeapons` has been added. Happy to add another PR and change it at this point then (if not already done).